managed/data/iatemplates/agent_down.yml

```diff
@@ -0,0 +1,12 @@
+---
+templates:
+  - name: pmm_agent_down
+    version: 1
+    summary: PMM agent down
+    expr: 'pmm_managed_inventory_agents{agent_type="pmm-agent"} == bool 0 '
+    for: 1m
+    severity: critical
+    annotations:
+      description: |-
+        PMM agent on {{ $labels.node_id }} cannot be reached. Host may be down.
+      summary: PMM agent is down ({{ $labels.node_id }})
```

Comment:
Here, we either need to do the same, if not - then remove the `node_id` label altogether. Otherwise the user may interpret as being the `agent_id`, not the `node_id`.